apps/src/templates/teacherHomepage/CollapsibleSection.story.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import CollapsibleSection from './CollapsibleSection';
+import CourseCard from './CourseCard';
+import SetUpMessage from './SetUpMessage';
+
+const exampleCard = {
+  courseName: "CSP Unit 2 - Digital Information",
+  description: "Explore how more complex digital information is represented and manipulated through computation and visualization",
+  image: "this is where there will be the source for the photo",
+  link: "link to the course",
+  assignedSections: []
+};
+
+export default storybook => {
+  return storybook
+    .storiesOf('CollapsibleSections', module)
+    .addStoryTable([
+      {
+        name: 'Recent Courses - 2 courses ',
```

Comment:
Worth adding a story for 3 courses, and making sure that those beyond the first arent rendered?